sccyou

```diff
@@ -24,39 +24,29 @@ $(basename "${0}") ${VERSION}
    -o <dir>     write output files to the provided directory (default is
                 next to the input file)
    -s           make an srt output as well as scc
+   -w           make a webvtt (vtt) output as well as scc
    -y           overwrite any existing output files
-   -v           let ffmpeg be verbose, otherwise ffmpeg sdterr is hidden
+   -v           let ffmpeg be verbose, otherwise ffmpeg stderr is hidden
 
   Caption Discovery
-   The location of the captions may either be specified as a line of a
-   specified number of seconds may be searched to find captions. Note
-   that searching for the caption line takes time, so providing the '-l'
-   option to specify the line to use is faster.
+   The location of the captions may either be specified as a line or a
+   specified number of seconds may be searched to find captions.
 
    -l <line>    specify the line where caption data should be read,
                 otherwise the first ${SEARCH_FIRST_X_SECONDS_FOR_CAPTION_LINE} lines will be searched.
    -t <secs>    specify an amount of time in seconds for which to search
-                for captions, ${SEARCH_FIRST_X_SECONDS_FOR_CAPTION_LINE} is used by default. By default the first 20
-                seconds of the video is searched.
+                for captions, ${SEARCH_FIRST_X_SECONDS_FOR_CAPTION_LINE} is used by default.
 
   Outcome
-   will create MOVIE_FILE.scc and MOVIE_FILE.srt
-
-  Information about the scc format is available at http://www.theneitherworld.com/mcpoodle/SCC_TOOLS/DOCS/
```

Comment:
I found this helpful but ...

Reply:
I also find this useful. What about moving the info to the ReadMe file?